packages/core/src/_typography.scss

```diff
@@ -281,6 +285,38 @@ Styleguide preformatted
   @extend %code-block;
 }
 
+%keyboard {
+  $kbd-key-size: 25px;
+  display: inline-block;
+  border-radius: $pt-border-radius - 1;
+  box-shadow: $pt-elevation-shadow-2;
+  background: $white;
+  min-width: $kbd-key-size;
+  height: $kbd-key-size;
```

Comment:
@llorca this is a good time to fix #2453. what changes should I make? (or push them yourself)